aws_grant_user_access/src/process_event.py

```diff
@@ -76,11 +76,18 @@ def publish_sns_message(message: SNSMessage) -> None:
 
 
 def _validate_role(role_arn: str) -> Any:
-    PERMITTED_ROLES = ["engineer"]
+    PERMITTED_ROLES = [
```

Comment:
Can we make this a constant that we can just import in the test below rather than duplicating it?